source/contracts/reporting/Market.sol

```diff
@@ -265,7 +265,8 @@ contract Market is DelegationTarget, Typed, Initializable, Ownable, IMarket {
 
     function doFeePayout(bool _toOwner, uint256 _amount) private returns (bool) {
         if (_toOwner) {
-            require(getOwner().call.value(_amount)());
+            bool _success = getOwner().call.value(_amount)();
+            require(_success);
```

Comment:
Style: I would just do `require(getOwner().call.value(_amount)())` for all of these.

Though, in this case it looks like you already had that.  What made you decide to split this into two lines?

Reply:
Solium complaints